linux/Instructions_for_desktop_app.md

```diff
@@ -0,0 +1,4 @@
+# Instuctions for Setting Up your Desktop File For BlueBubbles in Linux
+1. edit the `Bluebubbles.desktop` change exec to the location of the `bluebubbles_app`
+2. edit the icon to the location of the icon usually in `flutter_assets`
```

Comment:
The icon will *always* be in flutter_assets and I've suggested updating that in the .desktop file. The main instruction should just be to change the path to the bundle in both Exec and Icon.